src/CLICommand/parameter.ts

```diff
@@ -40,24 +52,36 @@ export class Parameter implements ParameterData {
 		return !!this.parameterData.required;
 	}
 
-	public static declare(parameter: ParameterData): void {
+	public get forbiddenParametersInConjunction(): ParameterName[] {
+		return Array.isArray(this.parameterData.forbiddenConjunctionParameters)
+			? this.parameterData.forbiddenConjunctionParameters.slice()
+			: [];
+	}
+
+	public get requiredParametersInConjunction(): ParameterName[] {
+		return Array.isArray(this.parameterData.requiredConjunctionParameters)
+			? this.parameterData.requiredConjunctionParameters.slice()
+			: [];
+	}
+
+	public static declare(parameter: ParameterConfig): void {
 		Parameter.parameters.push(parameter);
 	}
 
 	/**
 	 * @name reset
 	 * For testing purposes only. It will just remove all parameters declaration
-	 * @returns {ParameterData[]} the removed parameters
+	 * @returns {ParameterConfig[]} the removed parameters
 	 */
-	public static reset(): ParameterData[] {
+	public static reset(): ParameterConfig[] {
 		return this.parameters.splice(0, this.parameters.length);
 	}
 
-	public static get(parameterName: ParameterName): ParameterData {
+	public static get(parameterName: ParameterName): ParameterConfig {
 		const param = Parameter.parameters.find((p) => p.name === parameterName);
 		if (!param) {
 			throw new Error(`No such parameter ${parameterName}`);
 		}
-		return param;
+		return Object.assign({}, param);
```

Comment:
```suggestion
		// It uses Object.assign in order to return a copy of the instance instead of the original one, so modifying it won't affect the original
		return Object.assign({}, param);
```